### ISSUE TITLE
certbot: error: unrecognized arguments

### ISSUE THREAD

**Issue by bhopmann** [helgeerbe/certbot-dns-ionos#5]
While using this command

`sudo certbot certonly --authenticator dns-ionos --dns-ionos-credentials /etc/letsencrypt/ionos.ini --dns-ionos-propagation-seconds 900 --server https://acme-v02.api.letsencrypt.org/directory --agree-tos --rsa-key-size 4096 -d my.domain.com`

I get the following error:
`certbot: error: unrecognized arguments: --dns-ionos-credentials /etc/letsencrypt/ionos.ini --dns-ionos-propagation-seconds 900`

**Comment by helgeerbe** [comment]
Is the plugin istalled?

What is the output of `certbot plugins` ?
I would expect this:
`* dns-ionos
Description: Obtain certificates using a DNS TXT record (if you are using IONOS
for DNS).
Interfaces: IAuthenticator, IPlugin
Entry point: dns-ionos = certbot_dns_ionos.dns_ionos:Authenticator
`

**Comment by helgeerbe** [comment]
Which version of cerbot are you using?

`certbot --version`
`certbot 1.11.0`

When I remember right the argument naming conventions changed latley in certbot

**Comment by bhopmann** [comment]
> Is the plugin istalled?

Sorry, no, it isn't. That was the crucial mistake!

> Which version of cerbot are you using?
`certbot --version`
`certbot 0.31.0`

**Comment by helgeerbe** [comment]
Version 0.31.0 is from February 2019. You should update your certbot version.

**Comment by clementmas** [comment]
When I run `certbot plugins` I don't see "dns-ionos" even though I ran `pip install certbot-dns-ionos`. So I get an "unrecognized arguments" error.

Is it because certbot is installed with snap?

**Comment by crehn** [comment]
I have the same issue. This is my setup:
* raspberry pi with raspberry pi os (Debian Bookworm)
* certbot was installed during the installation of nextcloudpi (i.e. the Debian package certbot)
* `sudo snap install certbot-dns-ionos`
* certbot 2.1.0

The plugin is not listed in the output of `certbot plugins`. It lists only `standalone` and `webroot`.

As I feared that the plugin is only known to the certbot that comes with the snap, I've also tried `/snap/certbot-dns-ionos/current/bin/certbot plugins` -- to no avail.

I've also tried the symlinks @clementmas spoke of. As `/usr/local/lib/python3.11/dist-packages` is empty, I've tried symlinking `/snap/certbot-dns-ionos/current/lib/python3.12/site-packages/certbot_dns_ionos`. Unfortunately that doesn't work either.
